src/datepicker/index.es

```diff
@@ -174,9 +179,18 @@ class Datepicker extends InputComponent {
   }
 }
 
+const Datepicker = ({ theme: passedTheme, ...props }) => {
```

Comment:
Just a baseline for discussion. Using hooks was the quickest way for me to implement this.